Makefile.tools

```diff
@@ -45,9 +46,13 @@ axe_build:
 	cd $(axe_dir)/src; ./make.sh
 	cp $(axe_dir)/src/axe $(BP_TOOLS_BIN_DIR)
 
+verible_build:
+	cd $(verible_dir)
+	bazel build -c opt  --//bazel:use_local_flex_bison :install -- $(BP_TOOLS_BIN_DIR)
```

Comment:
$(BAZEL) and add variable to Makefile.common

Reply:
I see.

Reply:
Done.